Plutarch/DataRepr.hs

```diff
@@ -124,3 +126,12 @@ instance PIsDataRepr a => PIsData (PIsDataReprInstances a) where
 
 instance PIsDataRepr a => PMatch (PIsDataReprInstances a) where
   pmatch x f = pmatchRepr (punsafeCoerce x) (f . PIsDataReprInstances)
+
+instance {-# OVERLAPPING #-} (Ledger.FromData h, Ledger.ToData h, PIsData p) => PLift h (PIsDataReprInstances p) where
```

Comment:
Do you have to mark this `OVERLAPPING` even though the other instance is `OVERLAPPABLE`?

Reply:
I recall the compiler forced me to (even though I know it should not be required), but both overlap pragmas are gone now in fa0b1d02e80bc69d4a3b5e3fd1d27aac8c3083ff